erdjs/src/smartcontracts/typesystem/typeMapper.ts

```diff
@@ -37,6 +37,7 @@ export class TypeMapper {
             // For the following open generics, we use a slightly different typing than the one defined by elrond-wasm-rs (temporary workaround).
             ["VarArgs", VariadicType],
             ["MultiResultVec", VariadicType],
+            ["variadic", VariadicType],
```

Comment:
begin with uppercase just like the rest of types?

Reply:
wasm ABI generator for smart contracts expose the MultiResultVec type as variadic